### ISSUE TITLE
Google Drive Clone Issue

### ISSUE THREAD

**Issue by KAMG95** [viperadnan-git/google-drive-telegram-bot#39]
When I use the command /copy and put the link I get this error 'googleapiclient.http - WARNING - Encountered 403 Forbidden with reason "userRateLimitExceeded"'
No matter the file was large or small I get the same error once I send the message

**Comment by viperadnan-git** [comment]
Try after 24 hours.

**Comment by KAMG95** [comment]
I have been trying for 3 days now and nothing changes

**Comment by SAOJSM** [comment]
Maybe you copy too big file, so you can't copy.
You can deploy gd-utils bot so that you can use service accounts to copy files.

> When I use the command /copy and put the link I get this error 'googleapiclient.http - WARNING - Encountered 403 Forbidden with reason "userRateLimitExceeded"'
> No matter the file was large or small I get the same error once I send the message

**Comment by KAMG95** [comment]
> 
> 
> Maybe you copy too big file, so you can't copy.
> You can deploy gd-utils bot so that you can use service accounts to copy files.
> 
The files were 200mb and the bot didn't copy them XD 
I will search about gd-utils bot thx

**Comment by KAMG95** [comment]
> 
> 
> Try after 24 hours.

Now the message changed to ' googleapiclient.http - WARNING - Encountered 403 Forbidden with reason "cannotCopyFile"'
Any fix???

**Comment by adityeah001** [comment]
> > Try after 24 hours.
> 
> Now the message changed to ' googleapiclient.http - WARNING - Encountered 403 Forbidden with reason "cannotCopyFile"'
> Any fix???

No.. u have to manually create a copy and then Downloads the file.. its drive side error.. not the bot

**Comment by jassujas** [comment]
How to create a gdrive bot
